### ISSUE TITLE
migration files not published under Laravel 5.4

### ISSUE THREAD

**Issue by iwebnaut** [Waavi/translation#89]
Under Laravel 5.4 database migration files are not published when installing

running this command:
php artisan vendor:publish only publishes config file

```
$ php artisan vendor:publish
Copied Directory [\vendor\laravel\framework\src\Illuminate\Not
ifications\resources\views] To [\resources\views\vendor\n
otifications]
Copied Directory [\vendor\laravel\framework\src\Illuminate\Pag
ination\resources\views] To [\resources\views\vendor\pagi
nation]
Copied File [\vendor\santigarcor\laratrust\src\config\config.p
hp] To [\config\laratrust.php]
Copied File [\vendor\santigarcor\laratrust\src\config\laratrus
t_seeder.php] To [\config\laratrust_seeder.php]
Copied Directory [\vendor\laravel\framework\src\Illuminate\Mai
l\resources\views] To [\resources\views\vendor\mail]m

Copied File [\vendor\waavi\translation\config\translator.php]
 To [\config\translator.php]
Publishing complete.
```

**Comment by Aliasay** [comment]
+1

**Comment by nermamax** [comment]
I have installed some packages that have validations in their vendor folders, previously one of the packages I used published these migrations to migrations folder by running:

php artisan vendor:publish only publishes config file

Now I found alteast 2 packages that no longer published migrations when running this command, so I went into vendor folder and grabbed migration file and manually moved it and when I run

composer dump-autoload I got a warrning message from it that said: Warning: Ambiguous class resolution, "CreateRevisionsTable" was found in both "$baseDir . '/database/migrations/2013_04_09_062329_create_revisions_table.php" and "C:\xampp\htdocs\example\vendor/venturecraft/revisionable/src/migrations\2013_04_09_062329_create_revisions_table.php", the first will be used.

why is this happening? Has something changed in 5.4?
